src/mame/apple/apple3_m.cpp

```diff
@@ -153,7 +159,7 @@ uint8_t apple3_state::apple3_c0xx_r(offs_t offset)
 
 		case 0x50: case 0x51: case 0x52: case 0x53:
 		case 0x54: case 0x55: case 0x56: case 0x57:
-			m_screen->update_partial(m_screen->vpos());
+			//m_screen->update_partial(m_screen->vpos());
```

Comment:
Is there some reason these are commented out?  That will pretty much disable screen mode splitting like Atomic Defense does.

Reply:
This does need some discussion. I did comment those out there and uncommented the update_partial that was in the scan-line callback. With the A3 running slower now, it really needs the screen to be draw a line at a time like the real raster would do. Without that change, there was artifacting on the screen during the smooth scroll. 

Its taking time to move all the memory for the lines that are scrolling, and it looks like it follows down behind the raster doing these changes so it has the full time to of a screen refresh period to do this.

Reply:
That's fine as long as nobody tries to do a mid-scan line split like the A2 French Touch demos.  Go ahead and just remove those calls instead of commenting them out then and we'll be good.

Reply:
Note that update_partial(scanline) is inclusive. I mean, if you're at the start or at the middle of the scanline, it will update the whole scanline. Maybe that's what you ran into.

Reply:
Thanks for explaining, understood now. I have uncommented these and kept them there now. I think I had thought it only needed to be updated in the scanline part.